app/api_alpha/endpoints/ogc/views.py

```diff
@@ -0,0 +1,414 @@
+                "tags": ["Capabilities"],
+                "responses": {
+                    "200": {
+                        "description": "Liste des collections",
+                    }
+                },
+            }
+        },
+        schemes=["ogc"],
+    )
+    def get(self, request: Request, *args, **kwargs):
+
+        data = {
+            "links": [
+                self._get_collections_link(request, is_currrent_page=True),
+                self._get_conformance_link(request),
+                self._get_root_link(
+                    request
+                ),  # Changed to a proper self-referencing root link
+            ],
+            "collections": [
+                self._get_buildings_collection(request),
+            ],
+        }
+        return Response(data)
+
+
+class OGCBuildingsCollectionView(OGCAPIBaseView):
+    @rnb_doc(
+        {
+            "get": {
+                "summary": "Métadonnées de la collection de bâtiments",
+                "description": "Fournit des informations détaillées sur la collection de bâtiments.",
+                "tags": ["Data"],
+                "responses": {
+                    "200": {
+                        "description": "Métadonnées de la collection",
+                    }
+                },
+            }
+        },
+        schemes=["ogc"],
+    )
+    def get(self, request: Request, *args, **kwargs):
+        # MODIFIED: Pass request to the helper
+        data = self._get_buildings_collection(request, is_currrent_page=True)
+        return Response(data)
+
+
+class OGCBuildingItemsView(OGCAPIBaseView):
+
+    renderer_classes = [GeoJSONRenderer, JSONRenderer]
+
+    @rnb_doc(
+        {
+            "get": {
+                "summary": "Liste des bâtiments",
+                "description": "Récupère une liste de bâtiments sous forme de FeatureCollection GeoJSON.",
+                "tags": ["Data"],
+                "parameters": [
+                    {
+                        "name": "bbox",
+                        "in": "query",
+                        "description": "Filtre géographique par Bounding Box.",
+                        "schema": {"type": "array"},
+                        "style": "form",
+                    },
+                    {
+                        "name": "insee_code",
+                        "in": "query",
+                        "description": "Filtre les bâtiments dont l'emprise au sol est située dans les limites géographiques de la commune ayant ce code INSEE.",
+                        "required": False,
+                        "schema": {"type": "string"},
+                        "example": "75101",
+                    },
+                    {
+                        "name": "limit",
+                        "in": "query",
+                        "description": "Nombre maximum de bâtiments à retourner dans la page de résultats. Valeur par défaut : 20. Valeur maximale : 100.",
```

Comment:
```suggestion
                        "description": "Nombre maximum de bâtiments à retourner dans la page de résultats.",
```

gitbook gère bien l'affichage des valeurs par défaut, pas la peine de le répeter je trouve.

<img width="590" height="196" alt="Capture d’écran 2025-09-29 à 15 08 10" src="https://github.com/user-attachments/assets/d925ba88-bad8-44bf-b339-a1395293e016" />

<img width="566" height="150" alt="Capture d’écran 2025-09-29 à 15 11 09" src="https://github.com/user-attachments/assets/47986930-0ca5-4827-b924-bc7945469935" />